lua/nvim-treesitter/query_predicates.lua

```diff
@@ -85,5 +86,22 @@ query.add_predicate('has-type?', function(match, pattern, bufnr, pred)
   return vim.tbl_contains(types, node:type())
 end)
 
+
+-- Inject a language based on the aliases from `config.injections.aliases`.
+-- Usage: (#inject! @_language [default])
+query.add_directive("inject!", function(match, pattern, bufnr, pred, metadata)
+  local match_id = pred[2]
```

Comment:
Can we extract most of this logic to a function that can be used by other directives? We may have other directives that do more complex injection resolution, but still will need to do alias look ups.